kernel/siw_cm.c

```diff
@@ -681,6 +723,39 @@ static int siw_proc_mpareq(struct siw_cep *cep)
 		cep->mpa.hdr.params.c = 1;
 #endif
 
+	cep->p2ptype = SIW_MPAV2_P2P_DISABLED;
+
+	if (__mpa_rr_revision(req->params.bits) == MPA_REVISION_2) {
+		if (req->params.bits & MPA_RR_FLAG_ENHANCED) {
```

Comment:
Changed in the whole file. It's just habit since it explicitly shows intention. But when in Rome...